Src/BootCamp.Chapter/TextTable.cs

```diff
@@ -38,7 +38,84 @@ public static class TextTable
         /// </summary>
         public static string Build(string message, int padding)
         {
-            return "";
+            if (message.Length == 0)
+            {
+                return ""; 
+            }
+            // make a array so I can find the longest word. 
+
+            var arrayOfWords = message.Split(Environment.NewLine);
+
+            // Find length of longest word 
+           var longestWord = ""; 
+
+            if (arrayOfWords.Length == 1)
+            {
+                longestWord = arrayOfWords[0]; 
+            }
+            else
+            {
+                switch (arrayOfWords[0].CompareTo(arrayOfWords[1]))
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use 'CompareOrdinal' or 'Compare' with the locale specified instead of 'CompareTo'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4929752)

Reply:
I could ignore these according to Kai in the channel